common/build/eslint-config-fluid/minimal-deprecated.js

```diff
@@ -3,6 +3,21 @@
  * Licensed under the MIT License.
  */
 
+/**
+ * Shared list of permitted imports for configuring and override the `import/no-internal-modules` rule.
+ */
+const permittedImports = [
+	// Within Fluid Framework, allow import of '/internal', '/beta', and 'alpha' exports.
+	"@fluidframework/*/beta",
+	"@fluidframework/*/alpha",
```

Comment:
I am pretty sure we do not want alpha imports in our codebase except for special cases.
I am not sure about utility of beta.
Internally I think we want public and internal in almost all cases.